app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/offline/OfflineDatabaseService.kt

```diff
@@ -248,12 +248,22 @@ class OfflineDatabaseService internal constructor(
                 firebaseUser.uid,
                 User.Type.FIREBASE,
                 firebaseUser.displayName,
-                firebaseUser.email
+                firebaseUser.email,
+                getUser(User.Type.FIREBASE, firebaseUser.uid)!!.isAdmin,
             )
         )
         return fromUser(userDao.load(firebaseUser.uid, User.Type.FIREBASE)!!)
     }
 
+    override suspend fun setAdminAccess(firebaseUser: FirebaseUser, adminAccess: Boolean): User {
+        throw Exception("No Admin In Offline Mode Error")
```

Comment:
Creating a custom exception or at least using `IllegalStateException` would be more descriptive